game-core/src/test/java/games/strategy/engine/message/RemoteActionCodeTest.java

```diff
@@ -14,33 +14,53 @@
 import org.junit.jupiter.params.aggregator.ArgumentsAggregator;
 import org.junit.jupiter.params.provider.CsvFileSource;
 
+/**
+ * Test class that verifies that remote operation codes didn't change unexpectedly due to some
+ * random change.
+ *
+ * <p>Please update {@code /required-op-codes.txt} accordingly in case the remote interfaces are
```

Comment:
A similar kind of comment is probably good to have on the annotation itself, to tell a person who is exploring adding that annotation to look here to add it to a test.

Reply:
I'm always a bit hesitant to make references to tests in non-test code, but a small note won't hurt